test/powershell/engine/ETS/Adapter.Tests.ps1

```diff
@@ -202,6 +226,20 @@ Describe "Adapter Tests" -tags "CI" {
         }
     }
 
+    Context "Init-only Property DotNetAdapter Tests" {
+        It "Init-only properties are settable during instantiation" {
+            $iotc = [InitOnlyTestClass]@{ Value = 123 }
+            $ioProperty = $iotc.psobject.Properties['Value']
+            $ioProperty.Value | Should -BeExactly 123l
+            $ioProperty.IsSettable | Should -BeFalse
+        }
+
+        It "Init-only properties are not settable via ETS" {
+            $iotc = [InitOnlyTestClass]@{ Value = 123 }
+            { $iotc.Value = 456 } | Should -Throw -ErrorId PropertyAssignmentException -ExpectedMessage "'Value' is a ReadOnly property."
```

Comment:
We can not use localized strings in tests.
```suggestion
            { $iotc.Value = 456 } | Should -Throw -ErrorId PropertyAssignmentException -ExpectedMessage "'Value' is a ReadOnly property."
```

Reply:
Gotcha. Should I just drop the `-ExpectedMessage` part completely, or do we have a way of testing localized error messages?

Reply:
We cannot use localized message in tests - it is not reliable. Please remove.